mindsdb/api/mysql/mysql_proxy/utilities/dump.py

```diff
@@ -94,14 +102,15 @@ def _dump_str(var: Any) -> str | None:
     Returns:
         str | None: The string representation of the value or None if the value is None
     """
-    if pd.isna(var):
+    if isinstance(var, list) is False and pd.isna(var):
```

Comment:
`pd.isna` should be called very often and could slow down mysql server, I think it works slower than regular type or value checks. 
can be a better way to do this checking?